src/bun.js/webcore/blob.zig

```diff
@@ -99,6 +99,12 @@ pub const Blob = struct {
     pub const SizeType = u52;
     pub const max_size = std.math.maxInt(SizeType);
 
+    /// According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date,
+    /// maximum Date in JavaScript is less than Number.MAX_SAFE_INTEGER (u52).
+    pub const JSTimeType = u52;
+    pub const MillisecPerSec = 1000;
```

Comment:
this code is no longer used so we can delete it

Reply:
oops, forget to remove.